### ISSUE TITLE
3D asset loading from links

### ISSUE THREAD

**Issue by jeanahelver** [Frooxius/NeosPublic#1352]
pasting from clipboard a link to a 3D asset file (FBX, gltf) pops up with the import dialog, but seems to always fail to import.

**Comment by Frooxius** [comment]
This technically not a bug. FBX, GLTF and such assets cannot be loaded by Neos, they need to imported, which converts them into Neos native formats. The importer library is designed to operate over a file system, so it cannot handle importing from the HTTP(s) and similar, since it's a different system.

Adding support for such imports is pretty complicated, as it would require either downloading the files first, but it's not clear which files are needed (e.g. many formats are composed from multiple files, rather than a single one, but which ones are needed is known by the importer) or creating some sort of wrapper to make the importer think it's interacting with a file system, which is also very involved.

It's something that could potentially be added, but considering the amount of effort required it's not going to be prioritized anytime soon, I recommend just downloading the file yourself first and then importing.

**Comment by mralext20** [comment]
this is currently possible via a workaround, using the static binary import ability to import from a http(s) url. 
![image](https://user-images.githubusercontent.com/4016295/99651408-07db1980-2a03-11eb-8629-4ba87318dbdb.png)

clicking on the file, then the import option works.

**Comment by Frooxius** [comment]
Yeah doing that will result in the file being downloaded first. However note that with this method some models won't import correctly that normally would, if they're composed from multiple files (e.g. external textures for FBX or formats like .OBJ)
